back/spec/acceptance/ideas_spec.rb

```diff
@@ -352,6 +352,20 @@
       end
     end
 
+    describe do
```

Comment:
Does this test take a long time to execute? If so, I would perhaps consider removing so CI doesn't take too long.

Reply:
yes i was checking if CI could take it but it can't. Removing it